.github/workflows/dea-intertidal-image.yml

```diff
@@ -30,13 +33,33 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: Build DEA Intertidal image
         timeout-minutes: 20
         shell: bash
         run: |
           docker-compose build
 
+      - name: Run tests
```

Comment:
This downloads tide data from Dropbox (to replace with S3 access to tide modelling files), then runs some tests using `docker compose run`. Then finally upload code coverage to `codecov`